mycroft/skills/diagnostics/dialog/en-us/drive.low.dialog

```diff
@@ -0,0 +1 @@
+{{mountpoint}} is running low it has {{free}} free it's used {{percent}}%.  You should delete some files, or move some files to another drive/partition.
```

Comment:
Also, despite the relevance of making some emphases to a low disk space, there is no need to advise users to delete files.